src/sentry/search/eap/spans/sentry_conventions/deprecated_attributes.json

```diff
@@ -0,0 +1,938 @@
+{
+  "_generated": "This file is generated. Do not modify it directly. See scripts/generate_deprecated_attributes_json.ts",
```

Comment:
Is this file being added separately? Should we merge it in first before landing this change

Reply:
Keeping previous comment to explain why; I think we should change the text here so its clearer that its from another Repo

https://github.com/getsentry/sentry-conventions/blob/main/scripts/generate_deprecated_attributes_json.ts